LayoutTests/imported/w3c/web-platform-tests/webidl/README.md

```diff
@@ -0,0 +1 @@
+Tests for the [Web IDL Standard](https://webidl.spec.whatwg.org/).
```

Comment:
These are currently imported in `LayoutTests/imported/w3c/web-platform-tests/WebIDL` (note case). Please remove the existing tests.